pkg/syncer/apis/edge/v1alpha1/edge-syncconfig.go

```diff
@@ -0,0 +1,78 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +crd
+// +genclient
+// +genclient:nonNamespaced
+// +kubebuilder:resource:scope=Cluster,shortName=esc
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type EdgeSyncConfig struct {
```

Comment:
BTW, you do not need "Edge" in the name of this type, it is not going in the same package as a TMC or any other syncer config.

Reply:
Thanks. I will do it when I consolidate syncer apis into `/pkg/apis`.